security/pam-u2f/Portfile

```diff
@@ -20,14 +21,19 @@ homepage            https://developers.yubico.com/pam-u2f/
 master_sites        https://developers.yubico.com/pam-u2f/Releases/
 distname            pam_u2f-${version}
 
-checksums           rmd160  16423970f8d802b8e4583e0f3bbc4054f24291f8 \
-                    sha256  52a203a6fab6160e06c1369ff104afed62007ca3ffbb40c297352232fa975c99 \
-                    size    384163
+checksums           rmd160  9bccdcc623a48767cd011d4d3d39f9b9c3ea3237 \
+                    sha256  b7d62340c4f49e19cca93a0d0f398e48befd3eea8f1d70cebb7f8b71f3bce38a \
+                    size    429822
 
-depends_build       port:pkgconfig
-depends_lib         port:libu2f-host port:libu2f-server
+depends_build       port:asciidoc port:autoconf port:automake port:gengetopt port:libtool port:pkgconfig
+depends_lib         port:libfido2
 
 configure.args      --with-pam-dir=${prefix}/lib/pam
+# Release 1.1.1 fails to build with clang, but clang works in the 1.1.2 pre-release.
+# The reason is due to a usage of the --wrap flag that only GCC's linker supports.
+# Consider removing these two lines when 1.1.2 is released.
+compiler.blacklist  *clang*
```

Comment:
I think we should wait for 1.1.2, or update to an older version without this problem.

Reply:
libu2f-server and libu2f-host were replaced by libfido2 after the 1.0.8 release. All releases afterwards need libfido2 version 1.3.0 or later. Just to be safe for the future, I also submitted a PR to update libfido2 to the latest 1.8.0 release. We can update your port to 1.1.0 if you want. It seems to work with Apple Clang.